### PR TITLE
New design

### PR DESCRIPTION
**Pull request by mtkocak**
To correct branch.

### CODE REVIEW

**Comment by mtkocak** [comment]
these two should happen together

https://github.com/cakephp/app/pull/252
https://github.com/cakephp/bake/pull/104

File: webroot/css/cake.css [hunk 1]
```diff
@@ -382,3 +382,175 @@ pre {
...
   margin-right: 9px;
 }
+
+/* Midori Universal */
+
+.top-bar.expanded .title-area {
+   background:#01545b;
+}
+
+.top-bar.expanded, .top-bar,.top-bar-section ul li,.top-bar-section li:not(.has-form) a:not(.button){
```
Comment (markstory) [inline webroot/css/cake.css]: Most of the other selectors are one per line.

File: webroot/css/cake.css [hunk 2]
```diff
@@ -382,3 +382,175 @@ pre {
...
+table thead tr{
+   border-bottom: 1px solid #1798A5;
+}
+
+table tr th{
+  padding: 0.5625rem 0.625rem;
+  font-size: 0.875rem;
+  color: #1798A5;
+  text-align: left;
+   border-bottom: 2px solid #1798A5;
```
Comment (markstory) [inline webroot/css/cake.css]: Indentation is inconsistent here.

File: webroot/css/cake.css [hunk 3]
```diff
@@ -382,3 +382,175 @@ pre {
...
+}
+
+#content{
+   padding:2rem;
+}
+
+#content h3{
+     color: #be140b;
+      padding-bottom:0.5rem;
+      margin-bottom:20px;
```
Comment (markstory) [inline webroot/css/cake.css]: Indentation is inconsistent here.

File: webroot/css/cake.css [hunk 4]
```diff
@@ -382,3 +382,175 @@ pre {
...
+    padding-bottom:0.5rem;
+    margin-top:20px;
+    margin-bottom:10px;
+    border-bottom:0px;
+}
+
+
+@media only screen and (min-width: 40.063em) {
+#actions-sidebar{
+   height:100vh;
```
Comment (markstory) [inline webroot/css/cake.css]: Neato.

Comment (ADmad) [inline webroot/css/cake.css]: :ok_hand:

Comment (bcrowe) [inline webroot/css/cake.css]: 👌

**Comment by mtkocak** [comment]
whoa. holy eyeglasses. I am going to fix it now.

File: src/Template/Layout/default.ctp [hunk 5]
```diff
@@ -34,26 +34,26 @@ $cakeDescription = 'CakePHP: the rapid development php framework';
...
-        <div class="header-title">
-            <span><?= $this->fetch('title') ?></span>
-        </div>
-        <div class="header-help">
-            <span><a target="_blank" href="http://book.cakephp.org/3.0/">Documentation</a></span>
-            <span><a target="_blank" href="http://api.cakephp.org/3.0/">API</a></span>
-        </div>
-    </header>
-    <div id="container">
+<nav class="top-bar expanded" data-topbar role="navigation">
```
Comment (ADmad) [inline src/Template/Layout/default.ctp]: Indentation is off.

File: webroot/css/cake.css [hunk 6]
```diff
@@ -382,3 +382,176 @@ pre {
   color: red;
   margin-right: 9px;
 }
+
+
+/* Midori Universal */
```
Comment (jadb) [inline webroot/css/cake.css]: What's Midori? A quick [google search](https://www.google.ca/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=midori%20universal) returned nothing.

**Comment by mtkocak** [comment]
Midori means green japanese :) check my website: http://www.mtkocak.com

File: webroot/css/cake.css (hunk 6 above)
Comment (AD7six) [inline webroot/css/cake.css]: This comment will confuse other users too, note that no other files contain "personal" comments like this.

File: src/Template/Layout/default.ctp [hunk 7]
```diff
@@ -1,17 +1,17 @@
...
+*
+* Licensed under The MIT License
+* For full copyright and license information, please see the LICENSE.txt
+* Redistributions of files must retain the above copyright notice.
+*
+* @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+* @link          http://cakephp.org CakePHP(tm) Project
+* @since         0.10.0
+* @license       http://www.opensource.org/licenses/mit-license.php MIT License
+*/
```
Comment (jadb) [inline src/Template/Layout/default.ctp]: Indentation is off.

**Comment by mtkocak** [comment]
Proud of it :)

**Comment by jadb** [comment]
nice job! :+1:

File: src/Template/Layout/default.ctp [hunk 8]
```diff
@@ -34,26 +34,24 @@ $cakeDescription = 'CakePHP: the rapid development php framework';
...
-            <?= $this->Flash->render() ?>
-
-            <div class="row">
-                <?= $this->fetch('content') ?>
-            </div>
-        </div>
-        <footer>
-        </footer>
-    </div>
+    <nav class="top-bar expanded" data-topbar role="navigation">
```
Comment (ionas) [inline src/Template/Layout/default.ctp]: Are there ways to do away with the inline classes? (also see below)

**Pull request merged**